.env.example

```diff
@@ -0,0 +1 @@
+NODE_AUTH_TOKEN=xxxx
```

Comment:
what's this? :-)

Reply:
If you want to publish locally you need this token to be set

Reply:
lied. Deleting. Not needed anymore